x-pack/test/stack_functional_integration/apps/ccs/ccs_discover.js

```diff
@@ -172,5 +174,34 @@ export default ({ getService, getPageObjects }) => {
         expect(hitCount).to.be('28,010');
       });
     });
+
+    it('should reload the saved search with persisted query to show the initial hit count', async function () {
+      await PageObjects.discover.selectIndexPattern('data:makelogs工程-*,local:makelogs工程-*');
+      // apply query some changes
+      await queryBar.setQuery('success');
+      await queryBar.submitQuery();
+      await retry.try(async () => {
+        const hitCountNumber = await PageObjects.discover.getHitCount();
+        const hitCount = parseInt(hitCountNumber.replace(/\,/g, ''));
+        log.debug('### hit count = ' + hitCount);
+        expect(hitCount).to.be.greaterThan(25000);
+        expect(hitCount).to.be.lessThan(28000);
+      });
+    });
+
+    it('should add a phrases filter', async function () {
+      await PageObjects.discover.selectIndexPattern('data:makelogs工程-*,local:makelogs工程-*');
+      const hitCountNumber = await PageObjects.discover.getHitCount();
+      const originalHitCount = parseInt(hitCountNumber.replace(/\,/g, ''));
```

Comment:
```suggestion
      const originalHitCount = parseInt(hitCountNumber.replace(/\,/g, ''), 10);
```

Sometimes javascript guesses the radix wrong. Explicitly specifying base 10 is a safer bet.